### DISCUSSION TITLE
Beginner in C, anybody help me real quick?

### DISCUSSION THREAD

**Discussion by kulsaj** [community/community#131747] [Programming Help] [answered]
### Body

Hey guys, i'm a complete noob when it comes to programming. I literally just started and was trying to do a little conversion table from fahrenheit to celsius as follows:

```
#include <stdio.h>



int main()
{int fahr;
for (fahr = 300; fahr >=0 , fahr-=20;) {
printf("%3d %6.1f\n", fahr, (5.0/9.0)*(fahr-32));
 }
}
```

It's in C language, and the output i get is this:
280  137.8
260  126.7
240  115.6
220  104.4
200   93.3
180   82.2
160   71.1
140   60.0
120   48.9
100   37.8
 80   26.7
 60   15.6
 40    4.4
 20   -6.7
which is not bad, but not what i thought i'd get. I wanted it to start at 300 and end at 0, not start at 280 and end at 20. Any idea of what i'm doing wrong? PS: i've already tried setting the condition to fahr>= -20 but it didn't work (still stopped at 20).
PS 2: i kind of discovered how to fix the start, i just changed fahr = 300 to fahr = 320. Not exactly what i wanted to do to make it work, but i guess it's okay.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment, marked as answer]
Hi, in your for loop condition you have a comma instead of a semicolon separating two expressions.

**Reply by kulsaj** [reply]
oh shit im so dumb for that, idk why i came up with that... anyways, tysm. Really appreciated.

**Reply by ebndev** [reply]
Hi @komoto-0, thanks for taking the time to answer! We appreciate you being so involved in the Community lately!

@kulsaj thanks for posting on Discussions as well. 😄 I'm going to close this discussion, as it was resolved.
